src/stan_math_backend/Stan_math_code_gen.ml

```diff
@@ -182,12 +177,19 @@ let pp_ctor ppf (p : typed_prog) =
     | _ -> None
   in
   let rec pp_statement_zeroing ppf s =
+    let pp_statement_list_zeroing ls =
+      (list ~sep:cut pp_statement_zeroing) ppf ls
+    in
     match s.stmt with
     | Decl {decl_id; decl_type; _} -> (
       match decl_type with
       | Sized st -> pp_set_size ppf (decl_id, st, DataOnly)
       | Unsized _ -> () )
-    | Block ls | SList ls -> (list ~sep:cut pp_statement_zeroing) ppf ls
+    | Block ls ->
+        pf ppf "{" ;
```

Comment:
I think pp_block below already adds the curly braces, no?

Reply:
Yes, it does in some places. But we need this to also get braces around the try catch blocks.

Reply:
Hmm I guess it makes sense. Seems slightly code smell that we're doing something for that here in the constructor for try catch rather than dealing with this in the try/catch level... Whatever, tests pass and we can deal with it later :)